lib/elixir/lib/calendar/time.ex

```diff
@@ -402,16 +402,16 @@ defmodule Time do
   ## Examples
```

Comment:
We should also update the description of this function. For example:

```suggestion
  The returned value is a two-element tuple with the number of seconds and microseconds.

  ## Examples
```